toml-test.el

```diff
@@ -1,3 +1,5 @@
+(add-to-list 'load-path (file-name-directory (or load-file-name buffer-file-name)))
```

Comment:
Is this line change necessary to run tests with `make test` ?
(It hasn't been required in my local environment or GitHub Actions so far)